lib/WP_Auth0_Options.php

```diff
@@ -13,10 +13,7 @@ public static function Instance() {
 	}
 
 	public function is_wp_registration_enabled() {
-		if ( is_multisite() ) {
-			return users_can_register_signup_filter();
-		}
-		return get_site_option( 'users_can_register', 0 ) == 1;
+		return is_multisite() ? users_can_register_signup_filter() : get_site_option( 'users_can_register' );
```

Comment:
would that second value be the same to `get_site_option( 'users_can_register', 0 ) == 1` (previous)? Is this tested somewhere?

Reply:
Common practice. `get_site_option( 'users_can_register', 0 ) == 1` means "get the option `users_can_register` and if it doesn't exist use `0` and then loosely compare that to `1`." 

If `users_can_register` is on, it will be a `"1"` (true), if it's off it will be a `"0"` (false), and if it's not set, `get_site_options()` will return an explicit `false`. [Codex](https://codex.wordpress.org/Function_Reference/get_site_option#Return_Values)

Reply:
thanks for that link ⚡️